source/_integrations/securitas_direct.markdown

```diff
@@ -19,40 +19,19 @@ To integrate Verisure with Home Assistant, add the following section to your `co
 
 Securitas Direct can be integrated by adding the following `securitas_direct` section to your `configuration.yaml` file:
 
-```yaml
-# Example configuration.yaml entry
-securitas_direct:
-  username: USERNAME
-  password: PASSWORD
-  installation: INSTALLATION_NUMBER
-  code: 1234
-  country: PT
-  lang: pt
-```
-
-{% configuration %}
-username:
-  description: Username for your Securitas Direct account.
-  required: true
-  type: string
-password:
-  description: Password for your Securitas Direct account.
-  required: true
-  type: string
-installation:
-  description: The number of your installation. To find information on installations run `pysecuritas -u USERNAME -p PASSWORD -c COUNTRY -l LANGUAGE INS`.
-  required: false
-  type: string
-country:
-  description: Country identification (PT, ES, FR, GB, IT,...)
-  required: false
-  type: string
-lang:
-  description: Message language (pt, es, fr, en, it,...)
-  required: false
-  type: string
-code:
-  description: PIN code to activate or deactivate alarm.
-  required: false
-  type: integer
-{% endconfiguration %}
\ No newline at end of file
+## Configuration
+
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’. Click the plus icon and type/select ‘Securitas Direct’.
+1. Choose your platform `Securitas Direct`.
+1. Enter your credentials and installation configuration.
+
+1. Click the Save button.
+
+| Attribute | Description | Required |
+| --------- | ----------- | ----------- |
+| `username` | Username for your Securitas Direct account.| `yes`
+| `password` | Password for your Securitas Direct account. | `yes`
+| `installation` | The number of your installation. To find information on installations run `pysecuritas -u USERNAME -p PASSWORD -c COUNTRY -l LANGUAGE INS`.| `yes`
+| `country` | Country identification (`PT`, `ES`, `FR`, `GB`, `IT`,...).| `default ES`
+| `lang` | Messages language (`pt`, `es`, `fr`, `en`, `it`,...).| `default es`
+| `code` | PIN code to activate or deactivate alarm.| `no`
```

Comment:
If you want to provide such a list, please use a configuration_basic block (examples can be found in Lovelace cards documentation in this repository).

Reply:
I was following this example: https://www.home-assistant.io/integrations/fireservicerota/